### ISSUE TITLE
Missing modules/packages in Run unit tests

### ISSUE THREAD

**Issue by martin-weber** [microsoft/MLOpsPython#267]
It looks as the step _Run unit tests_ in _Model CI Pipeline_ has not yet the packages from ci_dependecies.yml or conda_dependencies.yml applied.
If I get it right, the conda packages are first installed later by step the _Publish Azure Machine Learning Pipeline_ when creating the environment.

What is the best way to have the same modules/packages installed for the unit tests and for the later Environment?

**Comment by eedorenko** [comment]
The pipeline steps are running in a Docker container (mcr.microsoft.com/mlops/python:latest) which has all packages installed from ci_dependecies.yml

**Comment by martin-weber** [comment]
Thanks @eedorenko . If I understand that right, I need to [create a custom container](https://github.com/microsoft/MLOpsPython/blob/master/docs/custom_container.md) if I want to add dependencies in ci_dependencies.yml. Is that right?

**Comment by eedorenko** [comment]
Yes, @martin-weber, you understand that right.
-  You need to create a custom container if you want to add packages in ci_dependencies.yml and have them ready for my unit tests.
- Only ci_dependencies.yml dependencies is used in the image. conda_dpendencies.yml is used on AzureML compute
- Did you build/pushed your image and reconfigured the pipeline to use it?
- BTW mcr.microsoft.com/mlops/python:openhack has lightgbm installed

**Comment by martin-weber** [comment]
Thanks @eedorenko. Ok that solves my issue. 
- Will create a custom container. Was not sure if its really needed. Should we add a comment to ci_dependecies.yml that the container needs to be rebuilt?
- Yes, I used mcr.microsoft.com/mlops/python:openhack until I ran into this problem #265 respectively #266 and switched then to mcr.microsoft.com/mlops/python:latest that solved the #266 but lead to the issue described above.
